reference/performance/index-merge.md

```diff
@@ -0,0 +1,99 @@
+
+## Actual use cases
+
+`IndexMerge` allows the optimizer to use multiple indexes per table, and merge the results returned by the indexes before further operation.Take the query above as an example, the execution plans generated will be:
+
+```
++--------------------+-------+-----------+---------------------------------------------------------------+
+| id                 | count | task      | operator info                                                 |
++--------------------+-------+-----------+---------------------------------------------------------------+
+| IndexMerge_11      | 2.00  | root      |                                                               |
+| ├─IndexScan_8      | 1.00  | cop[tikv] | table:t, index:a, range:[1,1], keep order:false, stats:pseudo |
+| ├─IndexScan_9      | 1.00  | cop[tikv] | table:t, index:b, range:[1,1], keep order:false, stats:pseudo |
+| └─TableScan_10     | 2.00  | cop[tikv] | table:t, keep order:false, stats:pseudo                       |
++--------------------+-------+-----------+---------------------------------------------------------------+
+```
+
+The structure of the `IndexMerge` execution plan is similar to that of the `IndexLookUp` since both can be divided into index scans and full table scans. However, index scans of the `IndexMerge` may include multiple `IndexScan`. When the primary key index of the table is an integer, index scans may include one `TableScan`. For example:
+
+{{< copyable "sql" >}}
+
+```sql
+create table t(a int primary key, b int, c int, unique key(b));
+```
+
+```
+Query OK, 0 rows affected (0.01 sec)
+```
+
+{{< copyable "sql" >}}
+
+```sql
+explain select * from t where a = 1 or b = 1;
+```
+
+```
++--------------------+-------+-----------+---------------------------------------------------------------+
+| id                 | count | task      | operator info                                                 |
++--------------------+-------+-----------+---------------------------------------------------------------+
+| IndexMerge_11      | 2.00  | root      |                                                               |
+| ├─TableScan_8      | 1.00  | cop[tikv] | table:t, range:[1,1], keep order:false, stats:pseudo          |
+| ├─IndexScan_9      | 1.00  | cop[tikv] | table:t, index:b, range:[1,1], keep order:false, stats:pseudo |
+| └─TableScan_10     | 2.00  | cop[tikv] | table:t, keep order:false, stats:pseudo                       |
++--------------------+-------+-----------+---------------------------------------------------------------+
+4 rows in set (0.01 sec)
+```
+
+Note that `IndexMerge` is used only when optimizer use multiple indexes per table. If the condition in the query expression is `a = 1 and b = 1`, the optimizer uses the index `a` or the index `b`, instead of the `IndexMerge`, to access tables.
```

Comment:
```suggestion
Note that `IndexMerge` is considered only when optimizer cannot find single index access method for the table. If the condition in the query expression is `a = 1 and b = 1`, the optimizer uses the index `a` or the index `b`, instead of the `IndexMerge`, to access tables.
```